Buildings/Controls/OBC/ASHRAE/G36/Atomic/ActiveAirflowSetpointReheatBox.mo

```diff
@@ -0,0 +1,987 @@
+within Buildings.Controls.OBC.ASHRAE.G36.Atomic;
+block ActiveAirflowSetpointReheatBox
+  "Output the active airflow setpoint for VAV reheat terminal unit"
+
+  parameter Boolean occSen = true
+    "Set to true if the zone has occupancy sensor"
+    annotation(Dialog(group="Zone sensors"));
+  parameter Boolean winSen = true
+    "Set to true if the zone has window operation sensor and window is open"
+    annotation(Dialog(group="Zone sensors"));
+  parameter Boolean co2Sen = true
+    "Set to true if the zone has CO2 sensor"
+    annotation(Dialog(group="Zone sensors"));
+  parameter Modelica.SIunits.VolumeFlowRate VCooMax
+    "Zone maximum cooling airflow setpoint"
+    annotation(Dialog(group="Nominal condition"));
+  parameter Modelica.SIunits.VolumeFlowRate VMin
+    "Zone minimum airflow setpoint"
+    annotation(Dialog(group="Nominal condition"));
+  parameter Modelica.SIunits.VolumeFlowRate VHeaMax
+    "Zone maximum heating airflow setpoint"
+    annotation(Dialog(group="Nominal condition"));
+  parameter Modelica.SIunits.TemperatureDifference maxDt
+    "Zone maximum discharge air temperature above heating setpoint"
+    annotation(Dialog(group="Nominal condition"));
+  parameter Modelica.SIunits.VolumeFlowRate VMinCon
+    "VAV box controllable minimum"
+    annotation(Dialog(group="Nominal condition"));
+  parameter Modelica.SIunits.Area zonAre "Area of the zone"
+    annotation(Dialog(group="Nominal condition"));
+  parameter Real outAirPerAre(final unit = "m3/(s.m2)")=3e-4
+    "Outdoor air rate per unit area"
+    annotation(Dialog(group="Nominal condition"));
+  parameter Modelica.SIunits.VolumeFlowRate outAirPerPer=2.5e-3
+    "Outdoor air rate per person"
+    annotation(Dialog(group="Nominal condition"));
+  parameter Real co2Set = 894 "CO2 setpoints"
+    annotation(Dialog(group="Nominal condition"));
+
```

Comment:
what unit is co2set? Is it ppm? Please add in comment if it is ppm, or in `unit` if it is an SI unit.